HACKING.rst

```diff
@@ -86,18 +86,129 @@ You can also see all the environments by simply running ``tox list``
 Running ``tox run -m format`` and ``tox run -m lint`` before committing code is
 recommended.
 
-Code Style
-==========
-
 Commits
-#######
+-------
+
+Commit messages are based on the `conventional commit`_ style::
+
+  <type>(<optional scope>): <description>
+
+  <optional body>
+
+  <optional footer>
+
+The commit is divided into three sections: a header, body, and footer.
+
+Header
+======
+
+The header is required and consists of three subsections: a type,
+optional scope, and description. The header must be 72 characters or less.
+
+Types
+#####
+
+``build``
+"""""""""
+
+Commits that affect the build of an application or library.
+
+This includes dependency updates, which should use the ``deps`` scope
+(``build(deps):``).
```

Comment:
I'm fine with this, but we'll need to change the renovate config to use `build(deps)` rather than its current `chore(deps)`

Reply:
Agreed.  And I think that development dependencies should also be `build(deps)` as well for simplicity (even though they don't strictly affect the build).

Reply:
I updated renovate's config